### PR TITLE
Contribution page (for translators like that)

### PR DESCRIPTION
**Pull request by inouetakuya**
The Vue community in Japan is very active and has the contribution page which increases the community's motivation.

* https://jp.vuejs.org/contribution/

I make a proposal that we make the contribution page like the following to increase Nuxt translators motivation:

image

### CODE REVIEW

File: pages/contribution/_slug.vue [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      if (err.response.status !== 404) {
+        return error({ statusCode: 500, message: store.state.lang.text.an_error_occured })
+      }
+      return error({ statusCode: 404, message: store.state.lang.text.api_page_not_found })
+    }
+    data.attrs = res.data.attrs
+    data.body = res.data.body
+    data.docLink = `https://github.com/nuxt/docs/blob/master${path}.md`
+    if (store.state.lang.iso === 'ru') {
+      data.docLink = `https://github.com/translation-gang/ru.docs.nuxtjs/blob/translation-ru${path}.md`
```
Comment (pi0) [inline pages/contribution/_slug.vue]: Can we make it more dynamic? Maybe an object from API...

Comment (inouetakuya) [reply pages/contribution/_slug.vue]: Actually, this is the copy from the following like that and I don't know why ru and cn are special.

* https://github.com/inouetakuya/nuxtjs.org/blob/df6c19d2980b45dd610a37188f81dc5a2d375a52/pages/guide/_slug.vue#L40-L45

I have no good idea and I hope someone will refactor this in another pull request :sweat_smile:

**Comment by inouetakuya** [comment]
Ref: https://github.com/nuxt/docs/pull/561#issuecomment-381348085

I have added contributors section per page like the following:

image

**Comment by inouetakuya** [comment]
I will close this pull request because @Atinux has the better idea:

* https://github.com/nuxt/docs/pull/561#issuecomment-385256638

**Pull request closed without merging**